src/ui/public/render_directive/render_directive.js

```diff
@@ -6,7 +6,8 @@ module.directive('renderDirective', function () {
   return {
     restrict: 'E',
     scope: {
-      'definition': '='
+      'definition': '=',
+      'scope': '=?'
```

Comment:
What if the definition included a scope definition? We would have to implement that in render_directive, but that sounds like fun to me.

```
definition = {
  controller...
  controllerAs...
  scope: {
    hit: '=',
    indexPattern: '=',
    onSave: '&'
  }
}
```